### DISCUSSION TITLE
Welcome to this forum!

### DISCUSSION THREAD

**Discussion by westnordost** [streetcomplete/StreetComplete#2378] [General]
### Hello and welcome to the StreetComplete forum 👋

This is a place to give feedback about the app, ask questions, discuss and generally meet and mingle with other users of the app. 

Maybe there are other StreetComplete users or OpenStreetMap enthusiasts in your town? Or maybe you would like to discuss your (blue-sky) vision or inspiration of how the app may look in the future? Or you are confused about a thing the app does and would like to be eludicated about the details of it?
Then please do post here, you have come to the right place!

And remember ...
![image](https://user-images.githubusercontent.com/4661658/102422900-099afb80-4008-11eb-81a7-d1bf30c52860.png)
... and party on, dudes!

**Comment by AllotmentCyclist** [comment]
What is the quest for adding building types. Buildings without a type dont appear on the street complete Map. The quest to name them exists but is greyed out. Attempts to turn it on get a message saying other methods are available. But what are they???

**Reply by matkoniecz** [reply]
Building overlay. 

You can also accept message and enable it

**Reply by BaumiCoder** [reply]
@matkoniecz I have the same problem. I have the `Building` overlay enabled, but I still see no way to add the building types. So what other way to edit the building type, except for the Question `What kind of building is this?`, is there, which is `more effective and efficient to enter and update this data`?

**Reply by matkoniecz** [reply]
> but I still see no way to add the building types. 

what happens when you tap on building?

**Reply by BaumiCoder** [reply]
I have the `Building` overlay enabled and when I tap on a building in the map, I only get the default popup menu with `Create new note`, `Create new track recording` and `Open location in another app`. 

But what I found now is, that I can activate a single enabled layer with the button in the upper right corner, left to the hamburger menu. If I activate the `Building` layer there, all building are displayed in red and with a tap on a building I can choose the building type.

**Reply by matkoniecz** [reply]
>  If I activate the Building layer there

that is what I meant by "Building overlay enabled" - I am glad that it works!

what you meant by "Building overlay enabled"? If you would enable building type quest it should also appear (though maybe it was blocked by other nearby quests? Or it run into some other limit?)
